example/README.md

```diff
@@ -12,3 +12,14 @@
  * **shared_library** - demonstration of building a shared library and generating
    a header file for interop with C code.
  * **mix_o_files** - how to mix .zig and .c files together as object files
+ 
+## Run the examples
+to run these examples, use the latest zig, and a command like:
```

Comment:
If this section is helpful for people, I'd say it should be short, only suggest the easiest way, and link to external instructions for any details to avoid redundancy. I'd also encourage us to set an example of using proper capitalization and grammar in README prose (to the best of our ability).

```suggestion
To run an example, [build](https://github.com/ziglang/zig#building-from-source) or [download](https://ziglang.org/download) the latest version of Zig and call `zig run` on the example's [root file](https://ziglang.org/documentation/master/#Root-Source-File).

    zig run hello_world/hello_world.zig
```

(Pardon GitHub's lack of ability to render markdown within a suggestion.)

Reply:
I incorporated your main sentence in replacement of mine, as I agree it's good.  I didn't merge your suggestion as the zig run should be quoted.

Reply:
Gotcha, thanks. In markdown, four spaces actually does the same as triple backtick, although I know it's inconsistent with other Markdown code blocks in this repo. I wish GitHub's suggestion feature didn't screw up Markdown-in-Markdown.